plugins/catalyst/python/pyproject.toml

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+# Copyright (c) 2025 Munich Quantum Software Company GmbH
+# All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+#
+# Licensed under the MIT License
+
+[build-system]
+requires = [
+  "scikit-build-core>=0.11.1",
+  "setuptools-scm>=8.2",
+  "pybind11>=2.13.6",
+  "mqt-core>=3.0.0",
+  "pennylane @ https://test-files.pythonhosted.org/packages/a4/ae/b8b0fdcd91fe0e87d699d46938cfee0531e8093042dd9eff11abc247c02f/pennylane-0.42.0.dev27-py3-none-any.whl",
+  "pennylane-lightning @ https://test-files.pythonhosted.org/packages/4d/99/1cab1348e3b839f0c8df515dc6b2d347c5e913361ff65e6fa1f197c0a67a/pennylane_lightning-0.42.0.dev16-cp312-cp312-macosx_13_0_arm64.whl",
+  "pennylane-lightning-kokkos @ https://test-files.pythonhosted.org/packages/25/97/9707f73d2ed6e2549e4e099d0083481c144e984227e147b44cd2838535e6/pennylane_lightning_kokkos-0.42.0.dev16-cp312-cp312-macosx_13_0_arm64.whl",
+  "pennylane-catalyst @ https://test-files.pythonhosted.org/packages/00/1f/791910226d7363ba8272add71529c637426d1ac225a4e582ef8612b7ecee/pennylane_catalyst-0.12.0.dev37-cp312-abi3-macosx_13_0_arm64.whl",
+]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "mqt-catalyst-plugin"
+description = "The Munich Quantum Toolkit's Catalyst Plugin"
+readme = "../../../README.md"
+authors = [
+    { name = "Patrick Hopf", email = "patrick.hopf@tum.de" },
+    { name = "Yannick Stade", email = "yannick.stade@tum.de" },
+    { name = "Lukas Burgholzer", email = "burgholzer@me.com" }
+]
+keywords = ["MQT", "quantum-computing", "design-automation", "catalyst", "MLIR", "plugin"]
+license = "MIT"
+# we have to duplicate the license file here because scikit_build_core requires
+# "the pattern [specifying the license file] must match files within the project directory"
+license-files = ["LICENSE.md"]
```

Comment:
Given how we use a default MIT license, might we get around this limitation by simply not providing the `license-files` entry here?